searx/plugins/autodetect_search_language.py

```diff
@@ -1,97 +0,0 @@
-# SPDX-License-Identifier: AGPL-3.0-or-later
-# lint: pylint
-"""Plugin to detect the search language from the search query.
```

Comment:
The autodoc file of this source can also be deleted https://github.com/searxng/searxng/blob/master/docs/src/searx.plugins.autodetect_search_language.rst